### DISCUSSION TITLE
Meeting Agenda 2/27

### DISCUSSION THREAD

**Discussion by dyc3** [dyc3/senior-design#305] [Meeting Agendas/Minutes]
1. Review tasks from previous sprint
2. Sprint Planning
3. Meet with darian

**Comment by moreno-michael** [comment]
Meeting minutes 2/27

1. Task Review
- Issue #1339 in progress
- Issue #1360 in progress
- Issue #1358 in progress
2. Sprint Planning
- Issue #1360, #1358, #1339 pushed to next sprint
- Issue #1338 assigned to Victor
- Issue #1337 reassigned to Chris
- More tasks to be assigned throughout the week
3. Met with Darian
- Expo info discussed
- Diagrams and tasks discussed
- Make sequence diagrams for issue #1339 and #1338
- Team dynamic issues discussed
